krun/vm_defs.py

```diff
@@ -293,6 +294,30 @@ def check_benchmark_files(self, benchmark, entry_point):
         if not os.path.exists(classfile_path):
             fatal("Benchmark file non-existent: %s" % classfile_path)
 
+
+def find_internal_jvmci_java_bin(base_dir):
+    """
+    The jvmci internal jdk8 seems to move around depending upon
+    the JVM with which it was built.
+
+    E.g. the java binary could be:
+    jvmci/jdk1.8.0-internal/product/bin/java
+
+    or it could be:
+    jvmci/jdk1.8.0_66-internal/product/bin/java
+
+    This is a helper function to try and find the 'java' binary
+    inside this "moving" directory.
+
+    arguments:
+    base_dir -- base jvmci directory"""
+
+    for f in os.listdir(base_dir):
```

Comment:
Could this be `for f in fnmatch.filter(os.listdir(base_dir), "jdk1.8*internal"):`?